sonar-project.properties

```diff
@@ -3,7 +3,7 @@ sonar.projectKey=SAP_open-ux-tools
 sonar.sources=.
 sonar.inclusions=**/*.ts, **/*.tsx
 sonar.exclusions=**/*.test.ts, **/*.test.tsx, **/*.story.tsx, **/examples/**/*, **/templates/**/*, **/test/**/*
-sonar.cpd.exclusions=**/i18n.ts, **/fiori-elements-writer/src/packageConfig.ts, **/fiori-freestyle-writer/src/packageConfig.ts
+sonar.cpd.exclusions=**/i18n.ts, **/fiori-elements-writer/src/packageConfig.ts, **/fiori-freestyle-writer/src/packageConfig.ts, **/ui5-info/src/ui5VersionFallback.ts
```

Comment:
Why is this excluded?

Reply:
Its essentially a json definition which inherently contains much replication. Please suggest how to remove otherwise as this seems appropriate to me.